src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -50,6 +52,28 @@ module _ {c ℓ} (S : Setoid c ℓ) where
   ∉-resp-≋ : ∀ {x} → (x ∉_) Respects _≋_
   ∉-resp-≋ xs≋ys v∉xs v∈ys = v∉xs (∈-resp-≋ (≋-sym xs≋ys) v∈ys)
 
+------------------------------------------------------------------------
+-- Irrelevance
+
+module _ {c ℓ} (S : Setoid c ℓ)
+               (≈-irrelevant : B.Irrelevant (Setoid._≈_ S)) where
```

Comment:
Again the irrelevance of the equality is important, could be moved to the proof? Apologies, I really should have spotted this before!

Reply:
:+1: